.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: ruby
+sudo: false
+before_install:
+  - gem install bundler -v 1.10
+bundler_args: --without development
+script:
+  - "bundle exec $CHECK"
+notifications:
+  email: false
+rvm:
+  - 2.1.9
+  - 2.3.3
```

Comment:
It's the two we build and bundle with puppet-agent. 2.1.9 is current, 2.3.3 will be switched to in Agent 5. Feel free to add more if you'd like.